tests/test_forms.py

```diff
@@ -59,6 +59,38 @@ def test_valid_emails(self, app, email_address):
             assert form.validate()
             assert form.data["test_email"] == email_address.strip()
 
+    def test_default_email_length(self, app):
+        email_address = 're' + 498 * 'a' + 'lly@long.com'  # 512 chars long
```

Comment:
Totally broken. The 'e' should be extended, not the 'a'.

Reply:
You're totally right. I can only apologise.

Have remedied.

Reply:
😂 👍